site-src/api-types/httproute.md

```diff
@@ -131,7 +131,8 @@ All filters are expected to be compatible with each other except for the
 URLRewrite and RequestRedirect filters, which may not be combined. If an
 implementation can not support other combinations of filters, they must clearly
 document that limitation. In all cases where incompatible or unsupported
-filters are specified, implementations MUST add a warning condition to status.
+filters are specified, implementations MUST add the `PartiallyInvalid` error
+condition with status `True` and reason `IncompatibleFilters`.
```

Comment:
Related to my comment above, this guidance doesn't really add up because the `PartiallyInvalid` condition is only intended for cases where a subset of rules are invalid, but here we're saying that it needs to be set _anytime_ there are incompatible filters.